### ISSUE TITLE
Cloudflare blocking access, changed user agent didn't work

### ISSUE THREAD

**Issue by squirtbrnr** [twrecked/hass-aarlo#787]
Starting sometime on 8/6 the integration went unavailable for me.  When I rebooted HA I get an authentication error in my logs and the aarlo integration fails to setup.  I regenerated my app password for my gmail account for 2fa.  I confirmed I could log into my Arlo account with the same credentials this integration is using.  I changed my aarlo config to a very bare bones config and it could not authenticate.  I'm enabling debug and rebooting and should have logs later.  What logs are needed?

```
This error originated from a custom integration.

Logger: pyaarlo
Source: custom_components/aarlo/pyaarlo/__init__.py:165
Integration: aarlo (documentation, issues)
First occurred: 10:23:48 PM (4 occurrences)
Last logged: 10:25:44 PM

authentication failed
```

My config:
```
aarlo:
  tfa_source: imap
  tfa_type: email
  tfa_host: imap.gmail.com
  tfa_username: !secret arlo_tfa_username
  tfa_password: !secret arlo_tfa_password
  username: !secret arlo_username
  password: !secret arlo_password
  reconnect_every: 90
```

**Comment by squirtbrnr** [comment]
after enabling debugging and looking at the logs, it appears I'm being blocked by Cloudflare, so this is a duplicate of https://github.com/twrecked/hass-aarlo/issues/782

**Comment by squirtbrnr** [comment]
except my version of the aarlo custom component is the latest release and the line `"Source": "arloCamWeb",` is already commented out.  Authentication is still failing

**Comment by squirtbrnr** [comment]
tried changing my user agent string in the config options to "random" per the suggestions and it still fails authentication with the cloudflare error in the logs.

**Comment by lowey71** [comment]
Reporting the same issue.  Using the commented out "Source".  Have not tried any other approaches

**Comment by squirtbrnr** [comment]
this issue already has an extensive thread.  See here: https://github.com/twrecked/hass-aarlo/issues/778

closing this in favor of watching the other thread.
